### DISCUSSION TITLE
Couldn't get invite to private room in Discord.

### DISCUSSION THREAD

**Discussion by rostialpin** [mrdbourke/zero-to-mastery-ml#54] [General]
I couldn't go through the verification even after logging in to Discord.  Please help to resolve the issue.
Thanks.
<img width="1728" alt="Screenshot 2023-08-01 at 4 19 21 PM" src="https://github.com/mrdbourke/zero-to-mastery-ml/assets/91621107/12ebce08-fc4a-44a1-ae5a-be27b38e70a9">

**Comment by mrdbourke** [comment]
Hey there! 

At the start of every ZTM course there's a lecture called "Meet your classmates and instructor".

And the link to the Discord is inside there! 

Let me know if you can find it.

Best,

Daniel

**Comment by KanikaGupta-22978** [comment]
I couldn't go through the verification even after logging in to Discord. Please help to resolve the issue.
Thanks.

**Reply by KanikaGupta-22978** [reply]
I just want the email id
registered mail id is as follows-ftskanika@gmail.com

**Comment by thekingslee** [comment]
I have a similar problem as well

**Comment by kRiShNa-429407** [comment]
It is made private for only ZTM students .

**Comment by Jng617** [comment]
If you're logged into Discord on your mobile app you need to scan the QR code or login via mobile to get in. It took me a few tries but that's how I was able to. Experienced the same issue as you
